app/build.gradle

```diff
@@ -61,9 +61,6 @@ android {
     }
 
     buildTypes {
-        debug {
-            testCoverageEnabled true
-        }
```

Comment:
Isn't this disabling test coverage? Why is this needed to fix the gRPC tests?

Reply:
Test coverage still runs without it, but I believe it will struggle to merge coverage from unit and instrumented tests. We were having some warnings related to it, but I just checked and they don't affect the tests passing, so I restored it.